src/storage/test/GetNeighborsBenchmark.cpp

```diff
@@ -0,0 +1,320 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include <folly/Benchmark.h>
+#include "fs/TempDir.h"
+#include "utils/NebulaKeyUtils.h"
+#include <gtest/gtest.h>
+#include <rocksdb/db.h>
+#include "storage/query/QueryBoundProcessor.h"
+#include "storage/test/TestUtils.h"
+#include "dataman/RowReader.h"
+#include "dataman/RowWriter.h"
+#include "meta/SchemaManager.h"
+#include "common/filter/Expressions.h"
+
+DEFINE_int64(max_rank, 1000, "max rank of each edge");
+DEFINE_double(filter_ratio, 0.5, "ratio of data would pass filter");
+
+std::unique_ptr<nebula::kvstore::KVStore> gKV;
+std::unique_ptr<nebula::storage::AdHocSchemaManager> gSchemaMan;
+std::unique_ptr<folly::CPUThreadPoolExecutor> gExecutor;
+
+namespace nebula {
+namespace storage {
+
+// only add two tag and two edge
```

Comment:
This comment is not right,
Here have 10 tags and 10,000 edges

Reply:
My bad.